cmd/syncthing/gui.go

```diff
@@ -1023,6 +1026,81 @@ func (s *apiService) getSystemLogTxt(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+type fileByte struct {
+	name string
+	buf  bytes.Buffer
+}
+
+func (s *apiService) getSupportBundle(w http.ResponseWriter, r *http.Request) {
+	var arrayFile []fileByte
+
+	jsonConfig, _ := json.MarshalIndent(getRedactedConfig(s), "", "  ")
+	arrayFile = addBufferFile(arrayFile, "config.json", jsonConfig)
+
+	jsonLog, _ := json.MarshalIndent(s.systemLog.Since(time.Time{}), "", "  ")
```

Comment:
I don't think we should drop in the log as JSON; a simple text format would be better and more readable. Basically, for each entry in the returned log lines, `fmt.Fprintf(buf, "%s: %s\n", entry.When, entry.Message)` (roughly) to just print it to a buffer and add that as the `log.txt` or something.